src/internal/components/LiveRegion/live-region-element/live-region-element.d.ts

```diff
@@ -0,0 +1,10 @@
+import {LiveRegionElement} from './element'
+import * as React from 'react'
+
+declare global {
+  namespace JSX {
+    interface IntrinsicElements {
+      'live-region': React.DetailedHTMLProps<React.HTMLAttributes<LiveRegionElement>, LiveRegionElement>
```

Comment:
Is this adding `<live-region>` to be identified as a JSX element? Or is it doing something else? I don't have a problem with this, I just want to educate myself.

Reply:
Yup, exactly 👍